script.js

```diff
@@ -0,0 +1,746 @@
+                }
+                app.state[region].laneData = app.makeLanesArray(region);
+            });
+            var url = "bikelanes/" + regions[app.display.regionsArray[0]].laneFiles.lanes;
+            app.laneAjaxQueue(url, app.display.regionsArray[0]);
+            console.log("  app.state: ", app.state);
+        }
+
+        // ======= clearAll =======
+        function clearAll() {
+            console.log("clearAll");
+
+            // == update clear/all button text
+            $('#clearAll-r > p').text('all');
+
+            // == hide buffer elements
+            var bufferEls = $('.td-b').children('.buffer');
+            bufferEls.each(function(index, bufferEl) {
+                $(bufferEl).css('visibility', 'hidden');
+            });
+
+            // == update region elements
+            var regionEls = $('div.region');
+            $(regionEls).each(function(index, regionEl) {
+                if ($(regionEl).hasClass('selected')) { $(regionEl).removeClass('selected') };
+            });
+
+            // == update buffer elements
+            var bufferEls = $('div.buffer');
+            $(bufferEls).each(function(index, bufferEl) {
+                if ($(bufferEl).hasClass('selected')) { $(bufferEl).removeClass('selected') };
+                if ($(bufferEl).hasClass('td-b')) {
+                    $(bufferEl).children().css('visibility', 'hidden');
+                }
+            });
+
+            // == clear lane and buffer states and data layers
+            $.each(app.state.selRegions, function(region, selected) {
+                if (selected) {
+                    app.state.selRegions[region] = false;
+                    app.state[region].laneLayers.forEach(function (laneData) {
+                        app.activeMap.removeLayer(laneData);
+                    })
+                    app.state[region].laneData = [];
+                    app.state[region].laneLayers = [];
+                    $.each(app.state[region].selBuffers, function(buffer, state) {
+                        buffer = false;
+                    });
+                    $.each(app.state[region].bufferLayers, function(buffer, bufferData) {
+                        if (bufferData) {
+                            app.activeMap.removeLayer(bufferData);
+                            bufferData = null;
+                        }
+                    });
+                }
+            });
+
+            // == restore regions array
+            app.display.regionsArray = ["AL", "AR", "DC", "MO", "PG"];
+            console.log("  app.state: ", app.state);
+        }
+    },
+
+    // ======= laneAjaxQueue =======
+    laneAjaxQueue: function(url, region) {
+        console.log("laneAjaxQueue");
+        var laneData = null;
+
+        $.ajax({
+            url: url,
+            method: "GET",
+            dataType: "text"
+        }).done(function(jsonData){
+            console.log("*** ajax success ***");
```

Comment:
Not sure what you're referring to in https://github.com/dcfemtech/hackforgood-waba-map/pull/78/files#r74975743 - are you saying that removing a `console.log` statement will disable the filter interface?